features/auth-features/org.wso2.carbon.auth.core.feature/resources/dbscripts/auth/h2.sql

```diff
@@ -24,6 +24,7 @@ CREATE TABLE IF NOT EXISTS AUTH_OAUTH2_APPLICATION (
             REDIRECT_URI VARCHAR(1024) DEFAULT NULL,
             GRANT_TYPES VARCHAR (1024) DEFAULT NULL,
             APP_STATE VARCHAR (25) DEFAULT 'ACTIVE',
+            TOKEN_TYPE VARCHAR (25) DEFAULT 'Default',
```

Comment:
did we change all the sqls?

Reply:
its not have all the sql scripts yet need to create track from 

https://github.com/wso2/carbon-auth/issues/141